lib/spreewald_support/version.rb

```diff
@@ -1,3 +1,3 @@
 module Spreewald
-  VERSION = '4.3.6'
+  VERSION = '4.4.6'
```

Comment:
Why not 4.4.0?

Reply:
You're right :) Thank you for noticing :)